tests/test_write_json.py

```diff
@@ -0,0 +1,298 @@
+"""test_write_json.py"""
+__author__ = "Marius Pahl"
+
+import json
+import pathlib
+import tempfile
+import unittest
+from collections import OrderedDict
+
+import pandas as pd
+
+from collect_stata.write_json import (
+    generate_statistics,
+    get_categorical_frequencies,
+    get_categorical_statistics,
+    get_nominal_statistics,
+    get_numerical_statistics,
+    get_univariate_statistics,
+    get_value_counts_and_frequencies,
+    write_json,
+)
+
+
+class TestWriteJson(unittest.TestCase):
+    """
+    Tests for all methods from write_json.py
+    """
+
+    def setUp(self):
+        """
+        Set a TemporaryDirectory for the json testfile
+        """
+        self.sandbox = tempfile.TemporaryDirectory()
+        return super().setUp()
+
+    @staticmethod
+    def get_testdatatable():
+        """
+        Create a test dataframe
+        """
+        input_path = pathlib.Path("tests/input")
+        test_data = pd.read_csv(input_path.joinpath("test_data.csv"))
+
+        return test_data
+
+    @staticmethod
+    def get_testmetadata():
+        """
+        Create test metadata
+        """
+        input_path = pathlib.Path("tests/input")
+        with open(input_path.joinpath("test_metadata.json")) as json_file:
+            test_metadata = json.load(json_file)
+
+        return test_metadata
+
+    @staticmethod
+    def get_teststudy():
+        """
+        Create a test study
+        """
+
+        return "teststudy"
+
+    def test_get_categorical_frequencies(self):
+        """
+        Test for get_categorical_frequencies
+        """
```

Comment:
This is redundant.
The information is already contained in the function name.
Describe what behavior the test is trying to check.
What is the intention?

Same applies to all following docstrings.